src/cpp/src/text_callback_streamer.hpp

```diff
@@ -11,12 +11,17 @@ namespace genai {
 
 class TextCallbackStreamer: public StreamerBase {
 public:
+    StreamerRunningStatus streaming_status = StreamerRunningStatus::UNDEF;
```

Comment:
as I see `StreamerBase` already contains this field ?

Reply:
removed